tdns/agent_structs.go

```
@@ -0,0 +1,195 @@
+}
+
+type AgentHelloResponse struct {
+	Status   string // ok | error | ...
+	Time     time.Time
+	Client   string
+	Msg      string
+	Error    bool
+	ErrorMsg string
+}
+
+type xxxAgentMsg struct {
```

golangci-lint (1.62.2)

type `xxxAgentMsg` is unused

(unused)